### PR TITLE
docs: add Optimizers naming to the README

### PR DESCRIPTION
**Pull request by julien-devatom**
# Pull Request

## Issue(s) fixed

This pull request fixes #to-complete

or

This pull request fixes:

- #to-complete
- #to-complete
- ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,40 +10,40 @@
 
 ---
 
-## What is Morpho?
+## What is Morpho Optimizers?
```
Comment (MerlinEgalite) [inline README.md:13]:
```suggestion
## What are Morpho Optimizers?
```

**Review by MerlinEgalite — DISMISSED** [review]

**Review by pakim249CAL — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -194,7 +193,7 @@ The report is then usable either:
 2 CI pipelines are currently running on every PR to check that the changes introduced are not modifying the storage layout of proxied smart contracts in an unsafe way:
 
 - [storage-layout.sh](./scripts/storage-layout.sh) checks that the latest foundry storage layout snapshot is identical to the committed storage layout snapshot
-- [foundry-storage-check](https://github.com/Rubilmax/foundry-storage-diff) is in test phase and will progressively replace the snapshot check
+- [foundry-storage-check](https://github.com/Rubilmax/foundry-storage-diff) is in the test phase and will progressively replace the snapshot check
```
Comment (MathisGD) [inline README.md:196]:
```suggestion
- [foundry-storage-check](https://github.com/Rubilmax/foundry-storage-diff) is in test phase and will progressively replace the snapshot check
```

File: README.md [hunk 3]
```diff
@@ -183,9 +182,9 @@ make coverage
 The report is then usable either:
 
 - via [Coverage Gutters](https://marketplace.visualstudio.com/items?itemName=ryanluker.vscode-coverage-gutters) following [this tutorial](https://mirror.xyz/devanon.eth/RrDvKPnlD-pmpuW7hQeR5wWdVjklrpOgPCOA-PJkWFU)
-- via html, using `make lcov-html` to transform the report and opening `coverage/index.html`
+- via HTML, using `make lcov-html` to transform the report and opening `coverage/index.html`
 
-:warning: Test coverage is not available on Morpho-AaveV2 for [this reason](https://github.com/foundry-rs/foundry/issues/3357#issuecomment-1297192171)
+:warning: Test coverage is not available on Morpho-AaveV2 Optimizer for [this reason](https://github.com/foundry-rs/foundry/issues/3357#issuecomment-1297192171)
```
Comment (MathisGD) [inline README.md:187]:
```suggestion
:warning: Test coverage is not available on the Morpho-AaveV2 Optimizer for [this reason](https://github.com/foundry-rs/foundry/issues/3357#issuecomment-1297192171)
```

File: README.md [hunk 4]
```diff
@@ -129,32 +128,32 @@ Refer to the `env.example` for the required environment variable.
...
 For the other commands, check the [Makefile](./Makefile).
 
-If you want to call a custom forge command, and not have to edit the `Makefile`, you can _source_ the `export_env.sh` script by calling `. ./export_env.sh`.
+If you want to call a custom forge command and not have to edit the `Makefile`, you can _source_ the `export_env.sh` script by calling `. ./export_env.sh`.
 
 :warning: The `export_env.sh` script exports environment variables in the current shell, meaning that subsequent calls to `make` or `forge` will use those variables. Variables defined in the `.env.local` file will still override those if you run `make` later. If you don't want to change variables in the current shell, you can always create a new shell in one of the following ways:
 
 - use `( . ./export_env.sh && forge test )` if the command you want to run is `forge test`
-- use `bash` and then `. ./export_env.sh` followed by your commands and then `exit` to return to the parent shell and clear the environment variables.
+- use `bash` and then `. ./export_env.sh` followed your commands and then `exit` to return to the parent shell and clear the environment variables.
```
Comment (MathisGD) [inline README.md:156]:
```suggestion
- use `bash` and then `. ./export_env.sh` followed by your commands and then `exit` to return to the parent shell and clear the environment variables.
```

File: README.md [hunk 5]
```diff
@@ -129,32 +128,32 @@ Refer to the `env.example` for the required environment variable.
...
 
-or to run only a specific set of tests of a specific protocol (e.g. for Morpho-Aave V2):
+or to run only a specific set of tests of a specific protocol (e.g. for Morpho-AaveV2 Optimizer):
 
 ```bash
 make c-TestBorrow PROTOCOL=aave-v2
 ```
 
-or to run an individual test of a specific protocol (e.g. for Morpho-Aave V2):
+or to run an individual test of a specific protocol (e.g. for Morpho-AaveV2 Optimizer):
```
Comment (MathisGD) [inline README.md:143]:
```suggestion
or to run an individual test of a specific protocol (e.g. for the Morpho-AaveV2 Optimizer):
```

File: README.md [hunk 6]
```diff
@@ -129,32 +128,32 @@ Refer to the `env.example` for the required environment variable.
...
 
-To run every test of a specific protocol (e.g. for Morpho-Compound):
+To run every test of a specific protocol (e.g., for Morpho-Compound Optimizer):
 
 ```bash
 make test PROTOCOL=compound
 ```
 
-or to run only a specific set of tests of a specific protocol (e.g. for Morpho-Aave V2):
+or to run only a specific set of tests of a specific protocol (e.g. for Morpho-AaveV2 Optimizer):
```
Comment (MathisGD) [inline README.md:137]:
```suggestion
or to run only a specific set of tests of a specific protocol (e.g. for the Morpho-AaveV2 Optimizer):
```

File: README.md [hunk 7]
```diff
@@ -129,32 +128,32 @@ Refer to the `env.example` for the required environment variable.
 
 To run tests on different protocols, navigate a Unix terminal to the root folder of the project and run the command of your choice:
 
-To run every test of a specific protocol (e.g. for Morpho-Compound):
+To run every test of a specific protocol (e.g., for Morpho-Compound Optimizer):
```
Comment (MathisGD) [inline README.md:131]:
```suggestion
To run every test of a specific protocol (e.g. for the Morpho-Compound Optimizer):
```

File: README.md [hunk 8]
```diff
@@ -52,17 +52,16 @@ It also interacts with `RewardsManager`, which manages the underlying pool's rew
...
 A bug bounty is open on Immunefi. The rewards and scope are defined [here](https://immunefi.com/bounty/morpho/).
-You can also send an email to [security@morpho.xyz](mailto:security@morpho.xyz) if you find something worrying.
+You can email [security@morpho.org](mailto:security@morpho.org) if you find something worrying.
 
 ---
 
 ## Deployment Addresses
 
-### Morpho-Compound Ethereum
+### Morpho-Compound Ethereum Optimizer
```
Comment (MathisGD) [inline README.md:64]:
```suggestion
### Morpho-Compound Optimizer on Ethereum
```

File: README.md [hunk 9]
```diff
@@ -75,7 +74,7 @@ You can also send an email to [security@morpho.xyz](mailto:security@morpho.xyz)
 - Lens Extension: [0xc5c3bB32c70d1d547023346BD1E32a6c5BC7FD1e](https://etherscan.io/address/0xc5c3bB32c70d1d547023346BD1E32a6c5BC7FD1e)
 - CompRewardsLens: [0x9e977f745d5ae26c6d47ac5417ee112312873ba7](https://etherscan.io/address/0x9e977f745d5ae26c6d47ac5417ee112312873ba7)
 
-### Morpho-Aave-V2 Ethereum
+### Morpho-AaveV2 Ethereum Optimizer
```
Comment (MathisGD) [inline README.md:77]:
```suggestion
### Morpho-AaveV2 Optimizer on Ethereum
```

File: README.md [hunk 10]
```diff
@@ -1,7 +1,7 @@
-# Morpho Core Protocol V1
+# Morpho V1 Optimizers Protocol
```
Comment (MathisGD) [inline README.md:1]:
```suggestion
# Morpho Optimizers Protocol
```

Comment (MathisGD) [reply README.md:1]: Or 
```suggestion
# Morpho Optimizers V1
```

Comment (MathisGD) [reply README.md:1]: Or
```suggestion
# Morpho Optimizers Protocol V1
```

File: README.md [hunk 11]
```diff
@@ -10,40 +10,40 @@
...
 
-TL;DR: Instead of borrowing or lending on your favorite pool like Compound or Aave, you would be better off using Morpho-Compound or Morpho-Aave.
+TL;DR: Instead of borrowing or lending on your favorite pool like Compound or Aave, you would be better off using Morpho-Compound or Morpho-Aave Optimizer.
 
 ---
 
 ## Contracts overview
 
-The Morpho protocol is designed at its core with a set of contracts delegating calls to implementation contracts (to overcome the contract size limit).
+The Morpho Optimizer protocol is designed at its core with a set of contracts delegating calls to implementation contracts (to overcome the contract size limit).
```
Comment (MathisGD) [inline README.md:26]:
```suggestion
Morpho Optimizers are designed at their core with a set of contracts delegating calls to implementation contracts (to overcome the contract size limit).
```

File: README.md [hunk 12]
```diff
@@ -10,40 +10,40 @@
 
 ---
 
-## What is Morpho?
+## What are Morpho Optimizers?
 
-Morpho is a lending pool optimizer: it improves the capital efficiency of positions on existing lending pools by seamlessly matching users peer-to-peer.
+Morpho optimizers improve the capital efficiency of positions on existing lending pools by seamlessly matching users peer-to-peer.
```
Comment (MathisGD) [inline README.md:15]:
```suggestion
Morpho Optimizers improve the capital efficiency of positions on existing lending pools by seamlessly matching users peer-to-peer.
```

File: README.md [hunk 13]
```diff
@@ -10,40 +10,40 @@
...
 
 ---
 
 ## Contracts overview
 
-The Morpho protocol is designed at its core with a set of contracts delegating calls to implementation contracts (to overcome the contract size limit).
+The Morpho Optimizer protocol is designed at its core with a set of contracts delegating calls to implementation contracts (to overcome the contract size limit).
 
-Here is a brief overview of the Morpho protocol's contracts interactions:
+Here is a brief overview of the Morpho Optimizers protocol's contracts interactions:
```
Comment (MathisGD) [inline README.md:28]:
```suggestion
Here is a brief overview of the Morpho Optimizers' contracts interactions:
```

File: README.md [hunk 14]
```diff
@@ -10,40 +10,40 @@
...
+The main user's entry points are exposed in the `Morpho` contract. It inherits from `MorphoGovernance`, which contains all the admin functions of the DAO, `MorphoUtils`, and `MorphoStorage`, where the protocol's storage is located. This contract delegates call to other contracts that have the same storage layout:
 
-- `PositionsManager`: logic of basic supply, borrow, withdraw, repay and liquidate functions. In Morpho-Aave, it is separated into two contracts, `EntryPositionsManager` and `ExitPositionsManager`. These contracts inherit from `MatchingEngine`, which contains the matching engine internal functions.
+- `PositionsManager`: logic of basic supply, borrow, withdraw, repay, and liquidate functions. Morpho-AaveV2 Optimizer is separated into two contracts, `EntryPositionsManager` and `ExitPositionsManager`. These contracts inherit from `MatchingEngine`, which contains the matching engine's internal functions.
```
Comment (MathisGD) [inline README.md:34]:
```suggestion
- `PositionsManager`: logic of basic supply, borrow, withdraw, repay, and liquidate functions. The Morpho-AaveV2 Optimizer is separated into two contracts, `EntryPositionsManager` and `ExitPositionsManager`. These contracts inherit from `MatchingEngine`, which contains the matching engine's internal functions.
```

File: README.md [hunk 15]
```diff
@@ -10,40 +10,40 @@
...
 
-- Morpho's rates stay between the supply rate and the borrow rate of the pool, reducing the interests paid by the borrowers while increasing the interests earned by the suppliers. It means that you are getting boosted peer-to-peer rates or, in the worst case scenario, the APY of the pool.
-- Morpho also preserves the same experience, the same liquidity and the same parameters (collateral factors, oracles, …) as the underlying pool.
+- Morpho's rates stay between the supply rate and the borrow rate of the pool, reducing the interest paid by the borrowers while increasing the interest earned by the suppliers. It means that you are getting boosted peer-to-peer rates or, in the worst-case scenario, the APY of the pool.
+- Morpho also preserves the same experience, the same liquidity, and the same parameters (collateral factors, oracles, …) as the underlying pool.
 
-TL;DR: Instead of borrowing or lending on your favorite pool like Compound or Aave, you would be better off using Morpho-Compound or Morpho-Aave.
+TL;DR: Instead of borrowing or lending on your favorite pool like Compound or Aave, you would be better off using Morpho-Compound or Morpho-Aave Optimizer.
```
Comment (MathisGD) [inline README.md:20]:
```suggestion
TL;DR: Instead of borrowing or lending on your favorite pool like Compound or Aave, you would be better off using Morpho-Compound Optimizer or Morpho-Aave Optimizer.
```
or 
```suggestion
TL;DR: Instead of borrowing or lending on your favorite pool like Compound or Aave, you would be better off using Morpho Optimizers.
```

File: README.md (hunk 10 above)
Comment (julien-devatom) [reply README.md]: +1 for # Morpho Optimizers V1

File: README.md [hunk 16]
```diff
@@ -230,7 +229,7 @@ make script-Deploy PROTOCOL=compound NETWORK=goerli SMODE=local
 
 ## Questions & Feedback
 
-For any question or feedback you can send an email to [merlin@morpho.xyz](mailto:merlin@morpho.xyz).
+For any questions or feedback, you can send an email to [merlin@morpho.org](mailto:merlin@morpho.org).
```
Comment (julien-devatom) [inline README.md:232]: Do we want to keep that or just redirect to the Issues section?

**Review by MathisGD — APPROVED** [review]

File: README.md (hunk 16 above)
Comment (MathisGD) [reply README.md:232]: IMO it's weird

**Review by MerlinEgalite — APPROVED** [review]
